cdc/owner/schema.go

```diff
@@ -246,20 +246,43 @@ func (s *schemaWrap4Owner) BuildDDLEvents(
 	return s.filterDDLEvents(ddlEvents)
 }
 
+// TODO: delete this function after integration test passed.
 func (s *schemaWrap4Owner) filterDDLEvents(ddlEvents []*model.DDLEvent) ([]*model.DDLEvent, error) {
 	res := make([]*model.DDLEvent, 0, len(ddlEvents))
 	for _, event := range ddlEvents {
-		ignored, err := s.filter.ShouldIgnoreDDLEvent(event)
-		if err != nil {
-			return nil, errors.Trace(err)
+		var (
+			ignored bool
+			err     error
+		)
+		if event.Type == timodel.ActionRenameTable {
+			ignored, err = s.filter.ShouldDiscardDDL(
+				event.StartTs,
+				event.Type,
+				event.PreTableInfo.TableName.Schema,
+				event.PreTableInfo.TableName.Table,
+				event.Query)
+			if err != nil {
+				return nil, errors.Trace(err)
+			}
+		} else {
+			ignored, err = s.filter.ShouldDiscardDDL(event.StartTs,
+				event.Type, event.TableInfo.TableName.Schema, event.TableInfo.TableName.Table, event.Query)
+			if err != nil {
+				return nil, errors.Trace(err)
+			}
 		}
 		if ignored {
 			s.metricIgnoreDDLEventCounter.Inc()
-			log.Info(
-				"DDL event ignored",
+			log.Panic(
```

Comment:
warning？

Reply:
I set it PANIC here for test purpose.  I will change it to ERROR level later.